packages/liotad/liotapkg.sh

```diff
@@ -95,9 +95,17 @@ then
     exit 6
 fi
 
+# Collect the supplied arguments and create a comma seperated string so that package manager can split using comma as
+# delimeter. The package name supplied could contain space character.
+comma_seperated_args="$1"
+shift
+for arg in "$@"; do
```

Comment:
i am not sure if i understand this. How does command line know start and stop of argument?

Reply:
$@ is an array containing command line arguments. The for loop in shell knows about start and end of an array.

Here, the first argument is copied and removed (using shift) from the beginning of the arguments array.  The remaining arguments are copied in a loop to created a comma separated string.

Reply:
Understood. Thanks for the changes. As discussed, please add an example(s) in Readme as to how to execute the load command to take care of spaces. All other code changes seem fine.

Reply:
Thanks, Piyush. I have included a note in the README